pallets/primitives/src/constants/mod.rs

```diff
@@ -1,22 +1,3 @@
-#![cfg_attr(not(feature = "std"), no_std)]
-
-//! A set of constant values used in runtime.
-//!
-//! Babe has slots within each epoch.
-//! Grandpa has sessions that are measured in blocks within each era.
-
-/// Money matters.
```

Comment:
i think we should keep all this below this line. i think you can use the `deposit` function for multisig (see standalone chain), and i've been using other current constants in other PRs